xarray/backends/api.py

```diff
@@ -1280,11 +1280,11 @@ def to_netcdf(
     if path_or_file is None:
         if engine is None:
             engine = "scipy"
```

Comment:
Question to other developers: should we switch the default to netcdf?

Possibly with a depreciation cycle?